libs/core/src/lib/date-picker/date-picker.component.html

```diff
@@ -39,7 +39,7 @@
                 [ngModel]="_inputFieldDate"
                 (keyup.enter)="handleInputChange($any($event.target).value, false)"
                 (ngModelChange)="handleInputChange($event, true)"
-                (blur)="handleInputChange($any($event.target).value, false)"
+                (blur)="onTouched(); handleInputChange($any($event.target).value, false)"
```

Comment:
create function `onBlur` and do `onTouched(); handleInputChange($any($event.target).value, false)`. Doing this on HTML side is not beautiful :D